pkg/image/prune/imagepruner.go

```diff
@@ -190,6 +190,7 @@ func addImageStreamsToGraph(g graph.Graph, streams *imageapi.ImageStreamList, al
 
 		glog.V(4).Infof("Adding ImageStream %s/%s to graph", stream.Namespace, stream.Name)
 		isNode := imagegraph.EnsureImageStreamNode(g, stream)
+		imagegraph.EnsureAllImageStreamTagsNode(g, stream)
```

Comment:
What info is missing from the stream to be able to create a fully populated ISTNode?